smaps/src/main/java/com/google/smaps/FileUpload.java

```diff
@@ -45,41 +46,35 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     // Resets the error message.
     errorMessage = "";
 
-    // Gets the file chosen by the user.
-    Part filePart = request.getPart("smapsFile");
+    // Check whether example mode was selected, if it wasn't selected the string will be null.
+    String exampleButton = request.getParameter("example-btn");
+    if (exampleButton != null) {
+      // Create an example input stream from the smaps-full file, and upload it.
+      File exampleFile = new File("../smaps-full.txt");
```

Comment:
Let's "deploy" this to test it out. I don't know for sure that ../smaps-full.txt will exist in the deployed environment or not (nor do I really grok how to get it there if it's not there already).